plugins/flexibleengine/src/main/java/org/eclipse/xpanse/orchestrator/plugin/flexibleengine/monitor/utils/FlexibleEngineMonitorConverter.java

```diff
@@ -102,9 +153,10 @@ public Map<String, MonitorResourceType> buildMonitorMetricUrls(DeployResource re
      */
     public Metric convertResponseToMetric(DeployResource deployResource,
                                           MonitorResourceType type,
-                                          ShowMetricDataResponse response) {
+                                          ShowMetricDataResponse response,
+                                          FlexibleEngineNameSpaceKind flexibleEngineNameSpaceKind) {
         Metric metric = new Metric();
```

Comment:
Metric class also has a new mandatory field called "unit". I see this is not being set in the FlexibleEngine plugin. 
Can you please check this? Ideally without this, the REST API method will not work at all.

Reply:
I have made changes, for both cpu and mem, the unit is '%'.